src/lib/cache/cache.js

```diff
@@ -52,11 +52,26 @@ class Cache {
      * See: https://redis.io/topics/pubsub
      */
     async connect() {
-        this.client = await this._getClient();
-        this.subscriptionClient = await this._getClient();
+        if (this._connected) {
```

Comment:
This could be an issue if the caller deliberately want to reconnect. He will not know nothing is happening when he calls this func. Why not throw an exception instead?

Reply:
done